modules/openxAnalyticsAdapter.js

```diff
@@ -119,10 +146,10 @@ function buildUtmLocalStorageKey(utmMarkKey) {
 }
 
 function getPublisherPlatformId() {
-  if (initOptions.publisherPlatformId !== undefined) {
-    if (typeof initOptions.publisherPlatformId === 'string') {
-      if (initOptions.publisherPlatformId !== '') {
-        return initOptions.publisherPlatformId;
+  if (analyticsConfig.publisherPlatformId !== undefined) {
+    if (typeof analyticsConfig.publisherPlatformId === 'string') {
+      if (analyticsConfig.publisherPlatformId !== '') {
+        return analyticsConfig.publisherPlatformId;
```

Comment:
helper functions ? function enforce_string_or_null(val,  key) ?

Reply:
We can remove these in the future.  It's overkill.